_sass/components/_header.scss

```diff
@@ -0,0 +1,72 @@
+.c-hero {
+  width: 100%;
+  height: 100%;
+  background: url('../assets/logo-background.svg'), $top-gradient, url("../assets/homepage-clouds.jpg");
+  background-repeat: no-repeat;
+  background-size: 800px, cover, 200%;
+  background-position: center 2px, center center, center bottom;
+  padding: 6rem 0 4rem;
+
+  @media (min-width:$bp-md) {
+    background-size: 1200px, cover, 150%;
+    background-position: center -4%, center center, bottom center;
+    padding: 8rem 0 4rem;
+  }
+
+  @media (min-width:$bp-lg) {
+    background-size: 1500px, cover, 100%;
+    background-position: center -4%, center center, center bottom;
+    padding: 12rem 0 4rem;
+  }
+
+  @media (min-width:$bp-xl) {
```

Comment:
This is great that you are building out your site mobile first! It's so hard to get into the mindset so this is awesome 🏆